met-web/src/components/publicDashboard/Dashboard.tsx

```diff
@@ -178,7 +178,7 @@ const Dashboard = () => {
                                                 engagementIsLoading={isEngagementLoading}
                                             />
                                         </Grid>
-                                        <Grid item sm={8} md={4} sx={{ width: isSmallScreen ? '90%' : '100%' }}>
+                                        <Grid item sm={8} md={4} sx={{ width: isTablet ? '90%' : '100%' }}>
```

Comment:
if we wanna change the width of this block why not change it using the grid points, like sm={7} instead of 8. Why change the width? it's kinda anti-pattern to do this